GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -248,6 +250,109 @@ void Renderer::renderCamera(std::shared_ptr<Scene> scene,
     }
 }
 
+void Renderer::renderCamera(std::shared_ptr<Scene> scene,
```

Comment:
It's a bit hard to be sure when I have to keep paging back and forth ... but it sure _looks_ like this rather complex method differs from the above in only a handful of the `glViewport()` calls. It seems it would be a lot more maintainable if both called a common method ....